dfetch/commands/list.py

```diff
@@ -24,14 +24,22 @@ class List(dfetch.commands.command.Command):
     @staticmethod
     def create_menu(subparsers: "argparse._SubParsersAction") -> None:
         """Add the parser menu for this action."""
-        dfetch.commands.command.Command.parser(subparsers, List)
+        parser = dfetch.commands.command.Command.parser(subparsers, List)
+        parser.add_argument(
+            "--project", "-p", action="append", help="list selected projects"
```

Comment:
Maybe make it a little more explicit?

```suggestion
            "--project", "-p", action="append", help="Only list selected projects"
```

Reply:
Manual updated in e645c0a9764a38ac0b188e38fd1c4a33059c0240.